modules/xfem/src/base/XFEM.C

```diff
@@ -225,6 +225,9 @@ XFEM::updateHeal()
 
   mesh_changed = healMesh();
 
+  if (mesh_changed)
+    buildEFAMesh();
```

Comment:
I'm a little nervous about what effects this changes would have. Maybe we should discuss.

Reply:
I cannot remember exactly why I added this. Maybe we need to rebuild the EFA mesh when the mesh is healed. If not, there might be some inconsistency issue.